corehq/ex-submodules/phonelog/__init__.py

```diff
@@ -0,0 +1,34 @@
+from architect.commands import partition
+
+from django.apps import AppConfig
+from django.db import ProgrammingError
+from django.db.models.signals import post_migrate
+
+
+def create_partitions(sender, **kwargs):
+    """
+    After running migrations, go through each of the models
+    in the app and ensure the partitions have been setup
+    """
+    paths = {model.__module__ for model in sender.get_models()}
+    for path in paths:
+        try:
+            partition.run(dict(module=path))
+        except ProgrammingError:
+            # Possibly because models were just un-migrated or
+            # fields have been changed that effect Architect
+            print("Unable to apply partitions for module '{}'".format(path))
+        else:
+            print("Applied partitions for module '{}'".format(path))
```

Comment:
I'm not sure. I just copied this from their docs so that I could quickly verify it and show you guys what it may look like. I'll do some more digging since it looks like we're all in agreement that this looks to be a good thing to move towards